app/src/mainwindow2.cpp

```diff
@@ -535,8 +563,27 @@ bool MainWindow2::openObject(QString strFilePath)
 
     Object* object = fm.load(strFilePath);
 
-    if (object == nullptr || !fm.error().ok())
+    if (!fm.error().ok())
```

Comment:
Is there a reason you're not checking for object being null anymore?

Reply:
Stroll down another ten lines or so. I had to put it in a separate if statement because fm.error() won't have a proper title, description and details if fm.error().ok() is true.

Reply:
Agh... my bad, totally missed that while reviewing 👍